grayskull/base/base_recipe.py

```diff
@@ -178,9 +179,24 @@ def generate_recipe(
 
         :param folder_path: Path to the folder
         """
-        recipe_dir = Path(folder_path) / self.get_var_content(
+        package_name = self["package"]["name"].values[0]
+        if origin_is_github(package_name):
+            package_name = package_name.split("/")[-1]
+
+        print(f"This is the package name: {package_name}")
+
+        package_name = "grayskull" #hardcoded
+
+        #recipe_dir = Path(folder_path) / self.get_var_content(
+        #    package_name
+        #)
+
+        """The above three lines of code have been commented out during testing"""
```

Comment:
We shouldn't need these lines at this point. If we do, it's a logic we need to fix before (in PyPI module).

```suggestion
```